sys/include/net/skald/eddystone.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_skald_ Skald meets Eddy
+ * @ingroup     net_skald
+ * @brief       Skald's Eddystone implementation
+ *
+ * # About
+ * This module allows for creation and advertisement of Eddystone beacons (see
+ * https://github.com/google/eddystone).
```

Comment:
Matter of test but maybe use a named link instead, for instance:

```markdown
This module allows for creation and advertisement of [Eddystone](https://github.com/google/eddystone)
beacons
```

Reply:
nope, like the written out link better.